config/bundles.php

```diff
@@ -16,6 +18,4 @@
     HWI\Bundle\OAuthBundle\HWIOAuthBundle::class => ['all' => true],
     Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
     Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle::class => ['dev' => true, 'test' => true],
-    Doctrine\Bundle\DoctrineCacheBundle\DoctrineCacheBundle::class => ['all' => true],
-    Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
```

Comment:
@GawainLynch sorry couldnt resist

Reply:
No worries, I wasn't sure if you'd flag that in the last review.

FWIW the reason I included them was they are/were added but Flex and my thought being "don't confuse people"

Reply:
Oh, and the "green" above … I am still suffering from Christmas it seems :rofl: